app/controllers/planning_application/consultations_controller.rb

```diff
@@ -157,5 +157,9 @@ def set_geojson_features
 
       @geojson_features = @consultation.polygon_search_and_boundary_geojson
     end
+
+    def letter_content
+      consultation_params[:neighbour_letter_content].gsub("# #{@consultation.neighbour_letter_header}/n/n", "")
+    end
```

Comment:
We show the letter as one piece of text to the user, but we send the header and the body text separately to Notify so that the header appears bigger. So we need to remove the header in the body content.